src/links.tsx

```diff
@@ -1,7 +1,7 @@
 import vironITLogo from "images/vironIT.png";
 import playgendary from "images/playgendary.png";
 import herocraft from "images/herocraft.png";
-import Home from "@/pages/home";
+import Home from "@/pages/home/home";
```

Comment:
on string 79 there is a TS error for Products component. Fix it

Reply:
this function was temporary unused, but error in it is now fixed